Library/Tales of the Solar Patrol/Races and Animals/Jupiter/Io/Rock-Men of Io.gct

```diff
@@ -0,0 +1,947 @@
+{
+	"type": "template",
+	"version": 2,
+	"id": "23a86764-8650-408b-892b-0a8c9444910f",
+	"advantages": [
+		{
+			"type": "advantage_container",
+			"id": "14f9dbb1-2a5a-4098-84ab-241de40bd3b1",
+			"name": "Rock-Men of Io",
+			"calc": {
+				"points": 0
+			},
+			"notes": "TSP 23",
```

Comment:
Page references should be in the page reference field and should not have spaces in them